src/NITTA/Model/Microarchitecture/Builder.hs

```diff
@@ -11,87 +10,21 @@ Maintainer  : aleksandr.penskoi@gmail.com
 Stability   : experimental
 -}
 module NITTA.Model.Microarchitecture.Builder (
```

Comment:
Renamed to Types. I didn't merge the module because Conf is about configuration files and this file is about DTOs for REST APIs. 

I think when this project will support multiple networks, it will be clearer what to do with the module